src/components/WordPanel/WordPanel.js

```diff
@@ -182,7 +182,7 @@ class WordPanel extends Component {
           <Clipboard
             button-type="button"
             className="btn btn-block btn-success mt-4"
-            data-clipboard-target=".text-pre"
+            data-clipboard-target=".clipboard-target"
```

Comment:
Note that it's important to find all instances where the CSS classes are used (with `ack` or `grep`). In this case, the clipboard library was relying on the class `text-pre` which no longer exists. I've replaced it with `clipboard-target` (see [here](https://github.com/perseids-project/greek-latin-demixer/pull/6/files#diff-5477f4438079ab5bb402a731b66537feL75) too).